### PR TITLE
NMS-13851: Fix NPE while loading topology

### PR DESCRIPTION
**Pull request by cgorantla**
Attempt to fix NPE while loading topology.

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13851

### CODE REVIEW

File: features/topology-map/plugins/org.opennms.features.topology.plugins.topo.linkd/src/main/java/org/opennms/features/topology/plugins/topo/linkd/internal/LinkdEdgeStatusProvider.java [hunk 1]
```diff
@@ -94,26 +95,27 @@ public String getNamespace() {
     }
 
     @Override
+    @Transactional
```
[the remark this answers is not in the document]
Comment (cgorantla) [reply features/topology-map/plugins/org.opennms.features.topology.plugins.topo.linkd/src/main/java/org/opennms/features/topology/plugins/topo/linkd/internal/LinkdEdgeStatusProvider.java]: yeah, overlooked, updated with SessionUtils

**Comment by cgorantla** [comment]
Think this should fix it but checking if I can reproduce original problem

**Review by aramosvizcarra — COMMENTED** [review]
Looks good to me, I'm just trying to replicate.

**Comment by cgorantla** [comment]
> Think this should fix it but checking if I can reproduce original problem

Modified query and tried, didn't reproduce the original problem. 
It may be some corner case.

**Review by RangerRick — APPROVED** [review]
looks reasonable to me

**Pull request merged**